src/main/java/org/opensearch/security/filter/NettyRequest.java

```diff
@@ -85,6 +89,18 @@ public Map<String, String> params() {
         return params(underlyingRequest.uri());
     }
 
+    @Override
+    public String param(String key) {
```

Comment:
nit: rename it to `consumeParameter`?

Reply:
Removing this new parameter, nothing to name here.  Resolving this thread.